db/questions.yml

```diff
@@ -59,103 +59,104 @@ which_room:
 kitchen_problem:
   question: What is the problem in your kitchen?
   answers:
-    - text: Cupboards
+    - text: Cupboards or Worktop
 
-    - text: Sink
-      next: is_sink_blocked
-
-    - text: Damp or mould
-      next: describe_kitchen_damp
-
-    - text: Taps
-      next: is_kitchen_tap_stuck_open
+    - text: Damp or Mould
+      next: describe_damp
 
     - text: Electrical
       next: kitchen_electrical_problem
 
-    - text: Other
-      next: kitchen_other_problem
+    - text: Heating
+      next: heating_problem
 
-is_kitchen_tap_stuck_open:
-  question: Is the tap stuck open?
-  answers:
-    - text: 'Yes'
-      page: emergency_contact
+    - text: Sink
+      next: sink_problem
+    
+    - text: Windows
+      next: window_problem
 
-    - text: 'No'
+    - text: Other
+      next: kitchen_other_problem
 
 kitchen_electrical_problem:
   question: Is your problem one of these?
   answers:
     - text: Extractor fan
       next: is_fan_leaking_water
 
-    - text: Lights
-      next: bathroom_lights_problem
-      todo: SHOULD GO SOMEWHERE ELSE?
+    - text: Light fitting
+      next: water_leaking_into_light
 
-    - text: Smoke detector
+    - text: Light switch
+      next: electrical_hazard_warning
+
+    - text: Smoke detector is beeping
+      page: emergency_contact
 
     - text: Sockets
       page: electrical_hazard_warning
 
-other_electrical_problem:
+    - Other: kitchen_other_problem  
```

Comment:
I'm guessing you were meant to write the following:

```ruby
  - text: Other
    goto: kitchen_other_problem
```

Reply:
hmm that's interesting, just checked my version I copied out of sublime and reads: 
    - Other: kitchen_other_problem